docs/package_templates/prebuilt_tool_package/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,13 @@
+from conans import ConanFile
+from conan.tools.build import cross_building
+
+
+# legacy validation with Conan 1.x
+class TestPackageV1Conan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+
+    def test(self):
+        if not cross_building(self):
```

Comment:
Why not using `can_run` here? Is it not available in the v1-v2 compatible syntax?

Reply:
Done, using can_run now.